package.json

```diff
@@ -89,6 +91,7 @@
     "@types/webpack-env": "1.13.4",
     "@types/zxcvbn": "4.4.0",
     "autodll-webpack-plugin": "0.3.9",
+    "bchaddrjs": "0.2.1",
```

Comment:
Note to self, this was fixed with https://github.com/trezor/trezor.js/pull/58 so I can remove this.